src/components/app/header/header.js

```diff
@@ -9,29 +10,66 @@ import ProfileImagePath from 'assets/images/profile.png';
 
 import './header.css';
 
+const docsDropdown = 'docsDropdown';
+const profileDropdown = 'profileDropdown';
+
+const parentHasClass = (element, ...searchClasses) => {
```

Comment:
This is sort of weird. If menu functionality was implemented and encapsulated in a component (or set of them), then this shouldn't be needed.

Reply:
True, but I'm building a reusable menu atm. See above comment.